bedrock/settings/static_media.py

```diff
@@ -1326,6 +1333,14 @@
         ),
         'output_filename': 'js/firefox_hello_ie9-bundle.js',
     },
+    'firefox-personal': {
+        'source_filenames': (
+            'js/libs/jquery-1.7.1.min.js',
```

Comment:
Instead of including jquery here, you should probably just include the common `firefox-resp` bundle in the page. Currently you're not loading other files, such as `global.js` and `global-init.js`, so you're not getting the download button JS etc.